crates/ruma-common/src/directory.rs

```diff
@@ -7,13 +7,16 @@ mod filter_room_type_serde;
 mod room_network_serde;
 
 use crate::{
-    room::RoomType, serde::StringEnum, OwnedMxcUri, OwnedRoomAliasId, OwnedRoomId, PrivOwnedStr,
+    room::{RoomSummary, RoomType},
+    serde::StringEnum,
+    OwnedMxcUri, OwnedRoomAliasId, OwnedRoomId, PrivOwnedStr,
 };
 
 /// A chunk of a room list response, describing one room.
 ///
-/// To create an instance of this type, first create a `PublicRoomsChunkInit` and convert it via
-/// `PublicRoomsChunk::from` / `.into()`.
+/// To create an instance of this type, first create a [`PublicRoomsChunkInit`] and convert it via
```

Comment:
We could actually remove `PublicRoomsChunkInit` now since it can be constructed from `RoomSummary`.